lib/functions/validation.sh

```diff
@@ -53,53 +53,39 @@ function GetValidationInfo() {
     eval "${VALIDATE_LANGUAGE}=${!VALIDATE_LANGUAGE,,}"
   done
 
-  ################################################
-  # Determine if any linters were explicitly set #
-  ################################################
-  ANY_SET="false"
-  ANY_TRUE="false"
-  ANY_FALSE="false"
-  # Loop through all languages
-  for LANGUAGE in "${LANGUAGE_ARRAY[@]}"; do
-    # build the variable
-    VALIDATE_LANGUAGE="VALIDATE_${LANGUAGE}"
-    # Check to see if the variable was set
-    if [ -n "${!VALIDATE_LANGUAGE}" ]; then
-      # It was set, need to set flag
-      ANY_SET="true"
-      if [ "${!VALIDATE_LANGUAGE}" == "true" ]; then
-        ANY_TRUE="true"
-      elif [ "${!VALIDATE_LANGUAGE}" == "false" ]; then
-        ANY_FALSE="true"
+  # Loop through linter packs to turn on any linters in those packs.
+  for PACK in "${LINTER_PACKS[@]}"; do
+    LINTERS="${PACK}[@]"
+    # Build the RUN_*_TOOLS variable
+    RUN_LINTER_PACK="RUN_${PACK}"
+
+    for LANGUAGE in "${!LINTERS}"; do
```

Comment:
Or should I call it `LANGUAGE_PACKS` to be consistent with their existing use of `LANGUAGE` (although it'd be more confusing to an end user -- e.g., what the hell's a 'security language pack'?).

Reply:
Yes I just feel let's keep it `LANGUAGE` what they have used just to maintain uniformity.